Update differences-classic-modern-search.md (#3148)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/differences-classic-modern-search.md

Search admin can customize the *classic* search experience, but not the Microsoft Search experience. As a search admin you can *tailor* Microsoft Search to your organization so it's easy for your users to find often needed content in your organization.

> For Example if your organization have Microsoft Search full deployed , Custom Result sources at site collection or tenant level won't affect the search result instead search admin can use Microsoft Search Verticarls [Learn about Manage search verticals](https://docs.microsoft.com/en-us/microsoftsearch/manage-verticals)

Reviewer: ```suggestion
For example, if your organization has Microsoft Search full deployed, custom result sources at site collection or tenant level won't affect the search result. The search admin can use Microsoft search verticals instead. [Learn about Manage search verticals](https://docs.microsoft.com/en-us/microsoftsearch/manage-verticals).
```
